specs/vm/instruction_set.md

```diff
@@ -730,6 +761,20 @@ Panic if:
 
 - `$is + imm * 4 > VM_MAX_RAM - 1`
 
+### JNZ: Jump if not zero
```

Comment:
Isn't `jnz $ra $rb` redundant to

```
jne $ra $zero $rb
```

Reply:
Ahhh you're right. The only reason that was needed was to free up 6 more bits of immediate. But without immediates, then it's redundant. Good catch!

Reply:
Fixed in [94b4982](https://github.com/FuelLabs/fuel-specs/pull/338/commits/94b49821328ec2379fbdac202819cca4c97a405a)